packages/connect/src/connector/specs/index.ts

```diff
@@ -1,5 +1,13 @@
+const mapNameToId: Map<string, string> = new Map<string, string>([
+  ["polkadot", "polkadot"],
+  ["ksmcc3", "kusama"],
+  ["rococo_v1_13", "rococo"],
+  ["westend2", "westend"],
+])
+
 export async function getSpec(chain: string): Promise<string> {
-  const specRaw = (await import("./generated/" + chain + ".js")) as
+  const fileName = mapNameToId.get(chain) || ""
+  const specRaw = (await import("./generated/" + fileName + ".js")) as
```

Comment:
And then here we can do:

```ts
export async function getSpec(chain: WellKnownChains): Promise<string> {
  const fileName = mapNameToId[chain]
```

which is simpler and more accurate.

Reply:
I wanted to avoid this as in polkadot repo the same names are used. If you think this is better, then i will proceed (and thus the mapping above will not be needed)

Reply:
We don't have to follow what the Polkadot repo does

Reply:
This is not needed anymore - since the naming of the files have altered